src/styles/cards.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  padding-top: 32px;
+  width: 166px;
```

Comment:
using `box-sizing: border-box` will allow you to avoid such magic numbers, please use that and set the width to 200px